docs/proposals/20210325-imagepredownload-during-inplace-update.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Proposal Template
+authors:
+- "@FillZpp"
+reviewers:
+- "@Fei-Guo"
+- "@furykerry"
+creation-date: 2021-03-26
+last-updated: 2021-03-26
+status: implementable
+---
+
+# Pre-download image during in-place update
+
+If CloneSet is going to update Pods in-place,
```

Comment:
this feature should not be limited to cloneset, adv statefulset , sidecarset can be implemented in the same way. So plz make proposal more generic.